mods/uxtheme-hook.wh.cpp

```diff
@@ -132,13 +140,13 @@ BOOL Wh_ModInit() {
     }
     
     // for logonui
-    Wh_SetFunctionHook((void*)SetSysColors, (void*)SetSysColors_hook,
+    WindhawkUtils::Wh_SetFunctionHookT((void*)SetSysColors, (void*)SetSysColors_hook,

```

Comment:
The point of `Wh_SetFunctionHookT` is that it verifies the types if you don't use casts. If you use casts, there's no point in using it.

BTW same for `WindhawkUtils::SYMBOL_HOOK`. You can remove the casts and if it doesn't compile, you have a problem with the types.

Reply:
ohh, ill revert it then

Reply:
Just remove the casts.